core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -544,6 +544,8 @@ private[chisel3] object ir {
         _.toFirrtl.flatMap(_.update(renames))
       )
 
+    // TODO this method doesn't compile with Scala3 because it
+    // conflicts with the built-in `copy` method
     def copy(
       name:        String = name,
       components:  Seq[Component] = components,
```

Comment:
This was added for backwards compatibility reasons when the classes in this file were [accidentally] public, this is private now so we can feel free to change it--it'd be best to just delete this, the extra constructor above, and the companion object below.

Reply:
(But that need not be part of this PR)

Reply:
ok i'll do it separately once this goes through